packages/next/next-server/server/render.tsx

```diff
@@ -329,6 +332,15 @@ export async function renderToHTML(
 
   const pageIsDynamic = isDynamicRoute(pathname)
 
+  if (
+    nextExportCommand &&
+    (hasPageGetInitialProps || defaultAppGetInitialProps)
+  ) {
+    warn(
+      `Detected getInitialProps on page ${pathname} while running "next export". It's recommended to use getStaticProps which has a more correct behavior for static exporting: https://err.sh/vercel/next.js/getinitialprops-export`
```

Comment:
The page linked here does not seem to exist, can you add it.

Reply:
@timneutkens Added that file in this PR